pyfr/solvers/aceuler/inters.py

```diff
@@ -0,0 +1,83 @@
+# -*- coding: utf-8 -*-
+
+from pyfr.solvers.baseadvec import (BaseAdvectionIntInters,
+                                    BaseAdvectionMPIInters,
+                                    BaseAdvectionBCInters)
+
+
+class ACEulerIntInters(BaseAdvectionIntInters):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+        self._be.pointwise.register('pyfr.solvers.aceuler.kernels.intcflux')
+
+        rsolver = self.cfg.get('solver-interfaces', 'riemann-solver')
+        tplargs = dict(ndims=self.ndims, nvars=self.nvars, rsolver=rsolver,
+                       c=self._tpl_c)
+
+        self.kernels['comm_flux'] = lambda: self._be.kernel(
+            'intcflux', tplargs=tplargs, dims=[self.ninterfpts],
+            ul=self._scal_lhs, ur=self._scal_rhs,
+            magnl=self._mag_pnorm_lhs, nl=self._norm_pnorm_lhs
+        )
+
+
+class ACEulerMPIInters(BaseAdvectionMPIInters):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+        self._be.pointwise.register('pyfr.solvers.aceuler.kernels.mpicflux')
+
+        rsolver = self.cfg.get('solver-interfaces', 'riemann-solver')
+        tplargs = dict(ndims=self.ndims, nvars=self.nvars, rsolver=rsolver,
+                       c=self._tpl_c)
+
+        self.kernels['comm_flux'] = lambda: self._be.kernel(
+            'mpicflux', tplargs, dims=[self.ninterfpts],
+            ul=self._scal_lhs, ur=self._scal_rhs,
+            magnl=self._mag_pnorm_lhs, nl=self._norm_pnorm_lhs
+        )
+
+
+class ACEulerBaseBCInters(BaseAdvectionBCInters):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+        self._be.pointwise.register('pyfr.solvers.aceuler.kernels.bccflux')
+
+        rsolver = self.cfg.get('solver-interfaces', 'riemann-solver')
+        tplargs = dict(ndims=self.ndims, nvars=self.nvars, rsolver=rsolver,
+                       c=self._tpl_c, bctype=self.type)
+
+        self.kernels['comm_flux'] = lambda: self._be.kernel(
+            'bccflux', tplargs, dims=[self.ninterfpts], ul=self._scal_lhs,
+            magnl=self._mag_pnorm_lhs, nl=self._norm_pnorm_lhs
+        )
+
+
+class ACEulerInflowBCInters(ACEulerBaseBCInters):
+    type = 'ac-in-fv'
+
+    def __init__(self, be, lhs, elemap, cfgsect, cfg):
+        super().__init__(be, lhs, elemap, cfgsect, cfg)
+
+        tplc = self._exp_opts(
+            ['u', 'v', 'w'][:self.ndims], lhs
```

Comment:
Replace the list with `'uvw'` which works because all of the variable names are one letter.  This should allow you to fit it all on one line.